instrumentation/rspec/lib/opentelemetry/instrumentation/rspec/version.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Instrumentation
+    module RSpec
+      VERSION = '0.0.0'
```

Comment:
>that makes sense to me, would then be 0.19.4-experimental, 0.0.1, or 0.0.1-experimental

I would go with `0.0.1`, or `0.0.1-experimental`. I don't actually have a really strong opinion on this it was just a loose suggestion, I was just thinking it might be nice to signal the uncertainty as this is likely to evolve.  

All that said, I think `0.0.1` is reasonable as technically none of the instrumentation gems have any stability guarantee until there's a formal spec for it and we 1.0.0 them.

Reply:
done